packages/layout/src/tree-layout/constants/defaults.js

```diff
@@ -0,0 +1,11 @@
+const DEFAULT_WIDTH = 600;
+const DEFAULT_HEIGHT = 400;
+const LAYOUT_ID = 'd2ad88af-7050-4c1c-b407-42745cfe3bd7';
```

Comment:
Layout Id is hard coded? Shouldn't it be unique everytime a new layout is created?